pkg/sandbox/fixtures/expanded_resolv.conf

```diff
@@ -0,0 +1,4 @@
+search 1.com 2.com 3.com 4.com 5.com 6.com 7.com
```

Comment:
can we clean this up after the test?

Reply:
I think we've already cleaned the output resolv.conf file up after each test.

https://github.com/cri-o/cri-o/blob/c53e78f45ffde7a3b7a3d93d9c8c9410ca0d4fec/pkg/sandbox/infra_test.go#L42-L43

Am I missing something?

Reply:
ah the test as it currently is is not quite right. If we do os.Remove(c.Path) after calling Expect(ParseDNSOptions...), it's possible ParseDNSOptions failed after creating the file but before creating. can you switch the order of the defer and Expect calls, and remove this file from the commit?

Reply:
> can you switch the order of the defer and Expect calls

That is a good point. Done.

> and remove this file from the commit?

I think this file is a test data as fixtures/resolv.conf is.

Should we have to refactor the test to be able to test without these files?
Let me know if that is what you request.

Reply:
for some reason I didn't see the fixtures directory when I looked before. We can have the refactor as a follow up (which I'd love if you wanted to take that on :smiley:  )